channeldb/channel.go

```diff
@@ -595,6 +600,37 @@ func (c *OpenChannel) MarkBorked() error {
 	return nil
 }
 
+// MarkCommitmentBroadcasted marks the channel as a commitment transaction has
+// been broadcast, either our own or the remote, and we should watch the chain
+// for it to confirm before taking any further action.
+func (c *OpenChannel) MarkCommitmentBroadcasted(closeType ClosureType) error {
```

Comment:
Was to make it possible to mark breach/remote close if we wanted that in the future. But since we can keep this backwards compatible by creating the `ChanStatus` field, I'll just have one state for now.